### ISSUE TITLE
aragonCLI install failing on clean Ubuntu install

### ISSUE THREAD

**Issue by chrishobcroft** [aragon/aragon-cli#549]
## 📚 Documentation

When following the [instructions to install aragonCLI](https://hack.aragon.org/docs/cli-intro) on a fresh Ubuntu install, the process fails with error:

`EACCES: permission denied, access '/usr/local/lib'`

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

Yes

### Steps to reproduce:

1. Install clean version of Ubuntu 19.04

2. Log in as an Administrator

3. Run `sudo apt install npm` to install npm (not mentioned in document, but suggested by Terminal)

4. Run `npm install -g @aragon/cli`

5. See the following reported in the console:

```
npm WARN npm npm does not support Node.js v10.15.2
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN checkPermissions Missing write access to /usr/local/lib
npm ERR! path /usr/local/lib
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib'
npm ERR!  { [Error: EACCES: permission denied, access '/usr/local/lib']
npm ERR!   stack:
npm ERR!    'Error: EACCES: permission denied, access \'/usr/local/lib\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib' }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator (though this is not recommended).
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/chrishobcroft/.npm/_logs/2019-06-22T16_31_40_110Z-debug.log
```

**Comment by 0xGabi** [comment]
Hey @chrishobcroft I think this is an issue with how you installed node. Check the [troubleshooting section for Mac](https://hack.aragon.org/docs/guides-faq#mac-considerations). And also this [stackoverflow answer](https://hack.aragon.org/docs/guides-faq#mac-considerations).

**Comment by chrishobcroft** [comment]
Thanks @0xGabi.

Can I perhaps suggest clarifying these instructions so that someone who is installing from e.g. a clean Ubuntu install, is able to easily take the steps required to get up and running?

I propose this, because there are a number of potential "power users" of Aragon's system who are *not* developers who are accustomed to configuring their systems using these tools.

If it's as difficult as I am finding it to get this working, then I fear that other similarly "semi-technical" people like myself are also struggling :)

**Comment by chrishobcroft** [comment]
Can someone confirm what the pre-requisites are for installing aragon cli?

I still can't get this to work on a clean install of Ubuntu 19.04...

**Comment by 0x6431346e** [comment]
> Hey @chrishobcroft I think this is an issue with how you installed node. Check the [troubleshooting section for Unix](https://hack.aragon.org/docs/guides-faq#mac-considerations). And also this [stackoverflow answer](https://hack.aragon.org/docs/guides-faq#mac-considerations).

The link to SO seems to be leading to hack.



> Can someone confirm what the pre-requisites are for installing aragon cli?
> 
> I still can't get this to work on a clean install of Ubuntu 19.04...

Other than node, that should be it.

> EACCES: permission denied, access '/usr/local/lib'

By default you need elevate privileges to write to `/usr/local/lib`.

There are a couple of solutions that people prefer:

1. [Install using nvm (node version manager)](https://stackoverflow.com/a/56695889/7793435) (recommended)
2. [Change the default location of global packages](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory) (my favorite)

3. [Run with sudo: `sudo npm i -g @aragon/cli`](https://stackoverflow.com/a/52455512/7793435) (I believe this is not a good practice as a package could be malicious)
4. [Change the owner of `/usr/local/lib`](https://stackoverflow.com/a/55048094/7793435) (this might be unsafe as well, I'm not sure)

**Comment by 0xGabi** [comment]
closed with #559
